channels/rdpdr/client/rdpdr_main.c

```diff
@@ -661,13 +671,18 @@ static UINT rdpdr_process_connect(rdpdrPlugin* rdpdr)
 	return error;
 }
 
-static void rdpdr_process_server_announce_request(rdpdrPlugin* rdpdr, wStream* s)
+static UINT rdpdr_process_server_announce_request(rdpdrPlugin* rdpdr, wStream* s)
 {
+	if (!rdpdr || !s)
```

Comment:
Same remark here, how would we call that static internal function with invalid parameters ? Perhaps it's not a so nice idea but I feel like the job should be done by the callers (perhaps that these tests aren't even necessary)